### ISSUE TITLE
[Sleeping] Remove license files from the repository

### ISSUE THREAD

**Issue by santisoler** [santisoler/cc-licenses#9]
Creative Commons make available all license texts on their GitHub repository: https://github.com/creativecommons/creativecommons.org

All translations are available in HTML files, while the English version of the license text is also available in plan text.
We should consider not hosting our own license texts and pointing to their files instead.
Until https://github.com/creativecommons/cc-licenses/issues/8 is fixed, we could start removing only the English version while keeping the translations.

**Comment by cyb3rko** [comment]
So if I understood right you want to remove those license texts from your repository which are officially published by CC?

**Comment by cyb3rko** [comment]
And the correct repository I think is https://github.com/creativecommons/cc-licenses.

**Comment by santisoler** [comment]
> So if I understood right you want to remove those license texts from your repository which are officially published by CC?

Yes! It's better to link to the original files that hosting our own. I haven't though about it before because I didn't know CC hosts the license text in plain text until last week. I obtained the license file from https://choosealicense.com/licenses/cc-by-4.0/ , but it's better to point people to the source.

> And the correct repository I think is https://github.com/creativecommons/cc-licenses.

No, that repository is where the issue I open has been moved to. But the license texts live inside https://github.com/creativecommons/creativecommons.org , under `docroot/legalcode`.

**Comment by cyb3rko** [comment]
> Yes! It's better to link to the original files that hosting our own. I haven't though about it before because I didn't know CC hosts the license text in plain text until last week. I obtained the license file from https://choosealicense.com/licenses/cc-by-4.0/ , but it's better to point people to the source.

Ok, I totally agree, this solution just makes sense :wink:.

> No, that repository is where the issue I open has been moved to. But the license texts live inside https://github.com/creativecommons/creativecommons.org , under `docroot/legalcode`.

You're right...

**Comment by cyb3rko** [comment]
Now, after #8 is merged, we can start here.

**Comment by deobald** [comment]
Given that the only files in `docroot/legalcode` are HTML, it seems unlikely GitHub will properly recognize them (though maybe that's changed since I last tried). If that _is_ still the case, I doubt I'm the only person who greatly appreciates this repo of plaintext CC licenses that GitHub does recognize. :) It would be a shame to lose it.

**Comment by santisoler** [comment]
Hi @deobald! Thanks for showing our appreciation for this repo. I agree that Creative Commons is not currently offering a proper alternative to the things we have in this repo, so I don't have any plans of shutting it down in the near future.

I opened this Issue to state the idea that having Creative Commons offering plain text versions of their licenses sounds much better than having them in a non-offiicial repository in GitHub: proper maintenance, better support for translations, more discoverability, official support, etc.

But until CC don't make this repo obsolete, I'll keep it alive and running. BTW, if that eventually happens, I'll probably keep this repo alive, put a note pointing people to the official versions and archive it (no new commits, issues or PRs).

So, you can all relax about the future of this repo.
